src/ess/dream/workflows/__init__.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2025 Scipp contributors (https://github.com/scipp)
+"""
+Workflow for the DREAM instrument.
+"""
+
+from .generic import DreamGenericWorkflow
+from .powder import (
+    DreamGeant4MonitorHistogramWorkflow,
+    DreamGeant4MonitorIntegratedWorkflow,
+    DreamGeant4ProtonChargeWorkflow,
+    DreamGeant4Workflow,
+    DreamPowderWorkflow,
+)
+
+__all__ = [
+    'DreamGeant4MonitorHistogramWorkflow',
+    'DreamGeant4MonitorIntegratedWorkflow',
+    'DreamGeant4ProtonChargeWorkflow',
+    'DreamGeant4Workflow',
+    'DreamGenericWorkflow',
+    'DreamPowderWorkflow',
+]
```

Comment:
Not sure why we need this subfolder. Confusing why some workflows are in `dream`, others only in `dream.workflows`. Why not have everything in `dream`? We won't have 1000 workflows so I think that should be fine?

Reply:
My comment was about which you have in the respective `__init__.py` on the various levels. Also, we should avoid too much user-visible nesting, since it is just complicated and confusing.